src/routes/videoPlaybackProxy.ts

```diff
@@ -108,8 +109,14 @@ videoPlaybackProxy.get("/", async (c) => {
         "last-modified": googlevideoResponse.headers.get("last-modified") || "",
     };
 
+    let status = googlevideoResponse.status;
+    if (requestBytes) {
+        status = 206;
+        headersForResponse["content-range"] = `bytes ${requestBytes}/*`;
+    }
```

Comment:
uses the format as described in https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/206

`bytes $start-$end/*`